src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -545,7 +545,7 @@ private string GetClientErrorMessage(Exception exception, string additionalInfor
             else
             {
                 return $"There was an unhandled exception on the current circuit, so this circuit will be terminated. For more details turn on " +
-                    $"detailed exceptions by setting 'DetailedErrors: true' in 'appSettings.Development.json' or set '{typeof(CircuitOptions).Name}.{nameof(CircuitOptions.DetailedErrors)}'. {additionalInformation}";
+                    $"detailed exceptions in '{typeof(CircuitOptions).Name}.{nameof(CircuitOptions.DetailedErrors)}'. {additionalInformation}";
```

Comment:
Make sure you don't undo https://github.com/aspnet/AspNetCore/pull/15156 when you merge this. I'm not sure if there is ore stuff like this, but this seemed suspicious. @javiercn